src/Symfony/Component/HttpFoundation/Session/Storage/Handler/PdoSessionHandler.php

```diff
@@ -237,6 +239,7 @@ public function createTable()
 
         try {
             $this->pdo->exec($sql);
+            $this->pdo->exec("CREATE INDEX EXPIRY ON $this->table ($this->lifetimeCol)");
```

Comment:
we could add this query (with default table+col) in UPGRADE4.4&5.0 files to hint ppl they should add the index